<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>947</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Backups</string>
   </property>
  </widget>
  <widget class="QComboBox" name="backup_combo">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>130</y>
     <width>129</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Backup Completo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Backup Incremental</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Backup diferencial</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>186</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Que tipo de backup desea realizar: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_backup_export">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Realizar backup</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>947</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Crear backup:</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>947</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Importar backup:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_backup_import">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Importar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
